docs/viam/_index.md

```diff
@@ -2,6 +2,7 @@
 title: "Viam in 3 minutes"
 linkTitle: "Viam in 3 minutes"
 description: "Viam is a complete software platform that supports every step of your robot development lifecycle."
+metaDescription: "Viam is a complete software platform for robots. It runs on 64-bit Linux and macOS, and provides services for computer vision, motion planning, SLAM, data management, and ML. You can make use of Viam's robot components and extend its capabilities with community-contributed and custom resources."
```

Comment:
```suggestion
metaDescription: "Viam is a complete software platform for robots. It runs on 64-bit Linux and macOS, and provides services for computer vision, motion planning, SLAM, data management, and machine learning. You can make use of Viam's pre-defined robot components and extend its capabilities with community-contributed and custom resources."
```

Reply:
- `pre-defined` - up to you: I think adds clarity to the value of `Viam's robot components` and why a Google Searcher might want to click this link. Or similar "out of the box" word if you've got a better one!